python/ray/tune/trainable.py

```diff
@@ -394,6 +394,90 @@ def restore_from_object(self, obj):
         self.restore(checkpoint_path)
         shutil.rmtree(tmpdir)
 
+    @staticmethod
+    def get_checkpoint_path(checkpoint_dir):
+        """Given a directory of a specific checkpoint, returns the full
+        checkpoint path.
+
+        Args:
+            checkpoint_dir (str): Path pointing to a checkpoint directory.
+        Returns:
+            A string with the full path to the checkpoint within the directory.
+        """
+        if 'checkpoint' not in checkpoint_dir:
+            raise ValueError("The given path does not point to a specific "
+                             f"checkpoint: {checkpoint_dir}")
+        # if it directly points to the checkpoint file, just return it
+        if os.path.isfile(checkpoint_dir):
+            return checkpoint_dir
+
+        # else derive the checkpoint file and return it
+        checkpoint_number = checkpoint_dir.split('_')[-1]
+        return os.path.join(checkpoint_dir, f'checkpoint-{checkpoint_number}')
```

Comment:
It seems like this tries to do something like `ray.tune.utils.trainable.TrainableUtils.find_checkpoint_dir()`.

Generally I think these functions should live in the TrainableUtil as well

Reply:
Ok, thanks for the hint. I wasn't available of that module yet. I'll move the code there.